src/core/message.ts

```diff
@@ -148,28 +148,40 @@ export interface Message {
   get updatedAt(): Date | undefined;
 
   /**
-   * Determines if the version of this message is older than the version of the given message.
+   * Determines if this message is an older version of the given message.
+   *
+   * **Note** that negating this function does not mean that the message is a newer
+   * version of the same message, as the two may be different messages entirely.
+   *
+   * ```ts
+   *  !message.isOlderVersionOf(other) !== message.isNewerVersionOf(other)
+   * ```
    * @param message The message to compare against.
-   * @returns true if the version of this message is before the given message.
-   * @throws {@link ErrorInfo} if both message serials do not match.
+   * @returns false if the messages are different or if this version is not older.
```

Comment:
```suggestion
   * @returns true if this version is older and the messages are the same
```

Document the `true` condition?

Reply:
done